src/main/java/org/fcrepo/migration/validator/Driver.java

```diff
@@ -68,10 +70,14 @@
             description = "Directory where cached index of datastreams (will reuse index if already exists)")
     private File indexDir;
 
-    @CommandLine.Option(names = {"--results-dir", "-r"}, order = 6,
+    @CommandLine.Option(names = {"--results-dir", "-r"}, defaultValue = "output", order = 6,
             description = "Directory where validation results are placed")
     private File resultsDirectory;
 
+    @CommandLine.Option(names = {"--ocfl-root-dir", "-r"}, order = 6,
```

Comment:
Maybe ensure that the `order` for each Option is unique?
`resultsDirectory` and `ocflRootDirectory` are both set to "6".